src/App.vue

```diff
@@ -1,59 +1,55 @@
 <template>
   <v-app>
     <!-- TODO: FIND A CLEANER WAY TO DO CONDITIONAL REDERING-->
-     <!-- <Sidebar /> -->
-      <Sidebar v-if="signedIn" v-on:signedOut="changeSignInState"/>
-      <v-content>
-        <div v-if="signedIn">
-          <div id="header">
-            <div id="header-left">
-            </div>
-            <div id="header-right">
-            </div>
-          </div>
-          <div class="container">
+    <!-- <Sidebar /> -->
+    <Sidebar v-if="signedIn" v-on:signedOut="changeSignInState" />
+    <v-content>
+      <div v-if="signedIn">
```

Comment:
Doesn't the v-if above already make up for this v-if ?

Reply:
If you're talking about the v-if for the div, that was branched from master.